cyberdrop_dl/main.py

```diff
@@ -13,6 +13,25 @@
 from cyberdrop_dl.utils.utilities import check_latest_pypi, log_with_color, check_partials_and_empty_folders, log
 from cyberdrop_dl.managers.console_manager import print_
 from cyberdrop_dl.clients.errors import InvalidYamlConfig
+from rich.console import Console
+from rich.logging import RichHandler
+
+DEFAULT_CONSOLE_WIDTH = 160
+
+RICH_HANDLER_CONFIG = { 
+    "show_time": True, 
+    "rich_tracebacks": True, 
+    "tracebacks_show_locals": False
+}
+
+RICH_HANDLER_DEBUG_CONFIG = {
+    "show_time": True, 
+    "rich_tracebacks": True, 
+    "tracebacks_show_locals": True,
+    "locals_max_string": DEFAULT_CONSOLE_WIDTH,
+    "tracebacks_extra_lines": 2,
+    "locals_max_length": 20
+}
 
```

Comment:
This is a partial implementation of the rich logger but is never created. Better to just rebase after #192 is merged

Reply:
I removed the lines shown because they are not needed
Also I deleted https://github.com/jbsparrow/CyberDropDownloader/pull/179, because it will be easier just to remake it